### ISSUE TITLE
Slider: broken styles in High Contrast in Edge Chromium

### ISSUE THREAD

**Issue by xugao** [microsoft/fluentui#13869]
### Please provide a reproduction of the issue in a codepen:
https://developer.microsoft.com/en-us/fluentui#/controls/web/slider

#### Actual behavior:
In Edge chromium:
1. part of the line is missing
2. cannot tell slider is disabled

#### Expected behavior:
See the expected behaviors in (old) Edge:
![image](https://user-images.githubusercontent.com/1207059/86174787-2170d880-bad7-11ea-9dbd-41020eb377ce.png)
